lib/organisms/gravity-form/_style.scss

```diff
@@ -0,0 +1,50 @@
+.ln-o-gravity-form {
+
+  .hidden {
+    display: none !important;
+  }
+
+  &-header {
```

Comment:
Is any of this actually required? I mean, are you going to put some default styles in here? I'm not sure you should because they will just be overridden by the app, unless they're related to functionality (like .hidden).